core-ui/src/components/Predefined/Details/PersistentVolumeClaims/PersistentVolumeClaims.details.js

```diff
@@ -0,0 +1,162 @@
+        entries={pvc.spec?.selector?.matchExpressions || []}
+        headerRenderer={headerRenderer}
+        rowRenderer={rowRenderer}
+        testid="daemon-set-tolerations"
+        i18n={i18n}
+        showSearchField={false}
+      />
+    </>
+  );
+};
+
+export const PVCConfiguration = pvc => {
+  const { t } = useTranslation();
+
+  const { data: storageClasses } = useGetList()(
+    '/apis/storage.k8s.io/v1/storageclasses',
+  );
+  return (
+    <LayoutPanel className="fd-margin--md" key={'pvc-configuration'}>
+      <LayoutPanel.Header>
+        <LayoutPanel.Head title="Configuration" />
+      </LayoutPanel.Header>
+      <LayoutPanel.Body>
+        <LayoutPanelRow
+          key={pvc.spec.volumeMode}
+          name={t('persistent-volume-claims.headers.volume-mode')}
+          value={pvc.spec.volumeMode}
+        />
+        <LayoutPanelRow
+          key={pvc.spec.accessModes}
+          name={t('persistent-volume-claims.headers.access-modes')}
+          value={<Tokens tokens={pvc.spec.accessModes} />}
+        />
+        <LayoutPanelRow
+          key={pvc.spec?.volumeName}
+          name={t('persistent-volume-claims.headers.volume-name')}
+          value={
+            pvc.spec?.volumeName ? (
+              <Link
+                onClick={() =>
+                  navigateToFixedPathResourceDetails(
+                    'persistentvolumes',
+                    pvc.spec?.volumeName,
+                  )
+                }
+              >
+                {pvc.spec?.volumeName}
+              </Link>
+            ) : (
+              <p>{EMPTY_TEXT_PLACEHOLDER}</p>
+            )
+          }
+        />
+        <LayoutPanelRow
+          key={pvc.spec.storageClassName}
+          name={t('persistent-volume-claims.headers.storage-class')}
+          value={
+            storageClasses?.find(
+              ({ metadata }) => metadata.name === pvc.spec.storageClassName,
+            ) ? (
+              <Link
+                onClick={() =>
+                  navigateToFixedPathResourceDetails(
+                    'storageclasses',
+                    pvc.spec?.storageClassName,
+                  )
+                }
+              >
+                {pvc.spec.storageClassName}
+              </Link>
+            ) : (
+              <p>
+                {pvc.spec.storageClassName !== ''
+                  ? pvc.spec.storageClassName
+                  : EMPTY_TEXT_PLACEHOLDER}
+              </p>
+            )
+          }
+        />
+      </LayoutPanel.Body>
+    </LayoutPanel>
+  );
+};
+
+export const PersistentVolumeClaimsDetails = ({
+  DefaultRenderer,
+  ...otherParams
+}) => {
+  const { t } = useTranslation();
+  const customColumns = [
+    {
+      header: t('subscription.headers.conditions.status'),
+      value: ({ status }) => (
+        <PersistentVolumeClaimStatus phase={status.phase} />
+      ),
+    },
+    {
+      header: t('persistent-volume-claims.headers.capacity'),
```

Comment:
Do we want to change the name of the value? I think we should stick to the naming the k8s gives us (so `storage` in this case)